--- dtopt-0.1/dtopt/tests.py.orig	2007-08-10 20:24:37.000000000 +0200
+++ dtopt-0.1/dtopt/tests.py	2023-10-28 21:55:30.314817194 +0200
@@ -19,6 +19,6 @@ See, you got an error!  Next:
 if __name__ == '__main__':
     import doctest
     doctest.testmod()
-    print 'Expect *one* error.  Zero is bad, two is bad.'
-    print 'One error is good'
+    print('Expect *one* error.  Zero is bad, two is bad.')
+    print('One error is good')
     
--- dtopt-0.1/dtopt/__init__.py.orig	2007-08-10 20:24:37.000000000 +0200
+++ dtopt-0.1/dtopt/__init__.py	2023-10-28 22:09:20.953650571 +0200
@@ -1,6 +1,9 @@
 import doctest
 import sys
 
+if sys.version_info[0] == 3:
+    basestring = str
+
 def install_option(option, after_unload=None):
     if isinstance(option, basestring):
         if option not in doctest.OPTIONFLAGS_BY_NAME:
